phpstan.neon.dist

```diff
@@ -1,15 +1,11 @@
 includes:
     - phpstan-baseline.neon
-    - vendor/phpstan/phpstan-doctrine/extension.neon
-    - vendor/phpstan/phpstan-webmozart-assert/extension.neon
-    - vendor/jangregor/phpstan-prophecy/extension.neon
-    - vendor/phpstan/phpstan-symfony/extension.neon
-    - vendor/phpstan/phpstan-symfony/rules.neon
 
 parameters:
     level: 6
 
     reportUnmatchedIgnoredErrors: false
+    treatPhpDocTypesAsCertain: false
```

Comment:
Same asthe comment below, maybe we can found a way to avoid to this (how? I don't know yet)